.github/workflows/check-submodule.yml

```diff
@@ -0,0 +1,44 @@
+name: check-submodule
```

Comment:
Maybe add a  comment explaining this is mostly a copy-paste of semgrep-proprietary/.../check-submodule.yml

Reply:
done.